cogboard-webapp/src/components/AppDialogContent.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
I think we don't need this kind of an abstraction, because AppDialog component already has the ability to render whatever you want there. You can abstract confirmation in other way and don't bind it to dialog for instance make a confirmation component which has 2 buttons and a text. If you want to make confirmation dialog component, you can make a component which renders AppDialog component with confirmation component etc.